p4-16/spec/P4-16-spec.mdk

```diff
@@ -4376,6 +4376,32 @@ only if they have the same element type and the same length.  Two
 stacks are equal if and only if all their corresponding elements are
 equal.  Note that the `nextIndex` value is not used in the equality comparison.
 
+### Header stack expressions { #sec-hs-init }
+
+An expression of the following form evaluates to a header stack: '('
```

Comment:
Should rephrase this in the same way as header initializers.
Add the cast to stacks as a legal cast.

Reply:
This is now addressed.